wicket-core/src/main/java/org/apache/wicket/util/tester/WicketTester.java

```diff
@@ -54,14 +38,24 @@
 import org.apache.wicket.request.mapper.parameter.PageParameters;
 import org.apache.wicket.util.lang.Args;
 import org.apache.wicket.util.lang.Objects;
+import org.opentest4j.AssertionFailedError;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.servlet.ServletContext;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Whild card imports should be expanded

Lot's of re-formatting here :( hard to get the changes ....

Reply:
Sorry, i will try to properly adjust the autoformatter

Reply:
i applied the autoformatter